multi-child-strawman.md

```diff
@@ -0,0 +1,289 @@
+```
+<ph id="dataType" ctype="x-python-brace-param">{dataType}</ph>
+```
+
+The next section describes the set of inline elements that will be available.
+
+## Proposal: Supported element types
+### Background
+
+Unfortunately XLIFF 1.2 and XLIFF 2.0 differ in which specific tags are used to represent different types of inline elements:
+
+#### XLIFF 2.0 inline elements
+
+- Standalone code (`<ph/>`) - Corresponds to a single position in the content (e.g. the `<br/>` tag in HTML)
+- Well-formed spanning code (`<pc></pc>`) - A code that wraps a section of content using a start and end marker (e.g. the `<b></b>` tag in HTML), that is properly nested (i.e. all spans are completely contained by other spans that wrap them)
+- Start marker of spanning code (`<sc/>`) - Start marker of a span that isn't properly nested or crosses segment boundaries ("orphan")
+- End marker of spanning code (`<ec/>`) - End marker of a span that isn't properly nested or crosses segment boundaries
+
+#### XLIFF 1.2 inline elements
+
+- Generic group placeholder (`<g></g>`) - Generic, well-formed spanning code.
+- Generic standalone placeholder (`<x/>`) - Generic isolated item in the content
+- Begin paired placeholder (`<bx/>`) - Start marker of a span that isn't properly nested
+- End paired placeholder (`<ex/>`) - End marker of a span that isn't properly nested
+- Placeholder (`<ph></ph>`) - Native well-formed spanning code
+- Begin paired tag (`<bpt></bpt>`) - Start marker of a paired sequence of native codes
+- End paired tag (`<ept></ept>`) - End marker of a paired sequence of native codes
+- Isolated tag (`<it></it>`) - Start or end marker of a paired sequence of native codes where the companion marker isn't in the translation unit
+- Sub-flow (`<sub></sub>`) - Sub-flow text inside a native code sequence (e.g. the `title` element of an HTML `<a>` tag) 
+
+
+Although the two versions define different elements, the underlying semantics of many of the elements are the same and they can be mapped to each other:
+
+| Use Case | Representation (1.2) | Representation (2.0) |
+| -------- | -------------------- | -------------------- |
+| **Generic** |
+| Standalone code | `<x/>` | `<ph/>` | 
+| Well-formed spanning code | `<g></g>` | `<pc></pc>` |
+| Start marker of spanning code | `<bx/>` | `<sc/>` |
+| End marker of spanning code | `<ex/>` | `<ec/>` |
+| Orphan start marker of spanning code | n/a | `<sc isolated="yes"/>` |
+| Orphan end marker of spanning code | n/a | `<ec isolated="yes"/>` |
+| **Native code** (same as generic for 2.0) | |  |
+| Well-formed spanning code | `<ph></ph>` | `<pc></pc>` |
+| Start marker of spanning code | `<bpt></bpt>` | `<sc/>` |
+| End marker of spanning code | `<ept></ept>` | `<ec/>` |
+| Orphan start marker of spanning code | `<it></it>` | `<sc isolated="yes"/>` |
+| Orphan end marker of spanning code | `<it></it>` | `<ec isolated="yes"/>` |
+| Sub-flow text | `<sub></sub>` | (separate `<unit></unit>`) |
+
+### Proposed implementation
```

Comment:
Option 1

I would say it’s ok to not support “Orphan” markers

Reply:
Since I first wrote this I have talked with the translation vendor that my company uses. They only "officially" support XLIFF 1.2. Also, their tools generally use `<ph></ph>` (the "native" span) rather than `<g></g>` (the generic span. Based on my conversations with them, I would personally prefer to go with Option 2. That way we can do a round-trip XLIFF 1.2 -> JS -> XLIFF 1.2 without losing the "native"-ness of the elements.

Are you okay with that @adrai?

Reply:
for sure... this is fine.